modules/installation-vsphere-infrastructure.adoc

```diff
@@ -48,5 +53,13 @@ You can host the VMware vSphere infrastructure on-premise or on a link:https://c
 
 [IMPORTANT]
 ====
-You must ensure that the time on your ESXi hosts is synchronized before you install {product-title}. See link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-8756D419-A878-4AE0-9183-C6D5A91A8FB1.html[Edit Time Configuration for a Host] in the VMware documentation.
+To ensure the best performance conditions for your cluster workloads that operate on {oci-first} and on the {ocvs-first} service, ensure volume performance units (VPUs) for your block volume is sized for your workloads.
+
+The following list provides some guidance in selecting the VPUs needed for specific performance needs:
+
+* Test or proof of concept environment: `100` GB, and `20` to `30` VPUs.
+* Basic environment: `500` GB, and `60` VPUs.
+* Heavy-use production environment: More than `500` GB, and `100` or more VPUs.
+
+Consider reserving additional VPUs to give enough capacity for updates and scaling activities.
```

Comment:
I am not sure if "reserving" is a common word use to in this context, considering VPU is an attribute of the disk which will provide more performance with additional costs.
I can see some alternatives:
- "Consider purchasing": more specific that the customer will pay more for it
- "Consider increasing"
- "Consider allocating"

let me know what do you think @racedo @makentenza

Reply:
https://docs.oracle.com/en-us/iaas/Content/Block/Concepts/blockvolumeperformance.htm
>  [...] you are purchasing 20 VPUs per GB/month[...] 
> [...] You can purchase more VPUs to allocate more resources to a volume[...]

Reply:
Thanks, Marco. "Purchasing" is usually a term exempt from customer docs as it's more towards the marketing side of the business. I can update to "increasing" if that works for everyone?